programs/drift/src/math/margin.rs

```diff
@@ -526,11 +526,13 @@ pub fn meets_withdraw_margin_requirement(
     }
 
     if calculation.get_num_of_liabilities()? > 1 {
-        validate!(
-            !calculation.with_isolated_liability,
-            ErrorCode::IsolatedAssetTierViolation,
-            "User attempting to increase number of liabilities above 1 with a isolated tier liability"
-        )?;
+        if !user.is_reduce_only() {
+            validate!(
+                !calculation.with_isolated_liability,
```

Comment:
if i have a perp and borrow, but want to withdraw my collateral, wont this throw?

Reply:
changed withdraw/place to have same req check

originally i thought it might be better to have stricter withdraw